bddtests/peer_basic.feature

```diff
@@ -10,7 +10,7 @@
 #@chaincodeImagesUpToDate
 Feature: lanching 3 peers
     As a HyperLedger developer
-    I want to be able to launch a 3 peers
+    I want to be able to launch 3 peers
```

Comment:
This is the only change making me nervous.  I am by no means a behave expert, but my understanding is this is a grammar change but I don't see any other hunks that correspond to this one when I would have expected this.  It's possible that I am either not seeing the corresponding update, or misunderstanding something fundamental.  Probably best if @jeffgarratt can comment on this particular hunk.